frontend/src/scenes/notebooks/Notebook/utils.ts

```diff
@@ -31,6 +31,8 @@ export type NotebookNodeAttributes<T extends CustomNotebookNodeAttributes> = T &
         expanded?: boolean
         showSettings?: boolean
     }
+    // TODO: Type this more specifically to be our supported nodes only
+    children?: NotebookNodeResource[]
```

Comment:
The only major thing for me is whether this should be the built in `content` that TipTap already supports. It feels like we're doing a lot around re-rendering & updating nodes that we would get for free if it leveraged TipTap.

Wondering if there's anything clever we can do with multiple TipTap editors to completely separate the two in the eyes of TipTap but connect them via logics for messaging.

Reply:
Definitely agree. I think this should be part of the next iteration - if we think this "canvas" ideas has a legs for the overall product - how do we build it the right way.

For now though I will keep it as is.